kernel/xpmem_mmu_notifier.c

```diff
@@ -65,10 +65,20 @@ xpmem_invalidate_PTEs_range(struct xpmem_thread_group *seg_tg,
  * XPMEM only uses the invalidate_range_end() portion. That is, when all pages
  * in the range have been unmapped and the pages have been freed by the VM.
  */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 0, 0)
```

Comment:
So, this 5.0.0 thing and not 5.3.0 ? just to confirm.

Reply:
Yes, for each change I looked back through the Linux releases to determine when the change was first introduced. I tested the patch on Ubuntu LTS with Linux 5.0.0-37-generic and 5.3.0-26-generic.